students/chan/westagram/users/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json
+from django.http      import JsonResponse, request
+from django.views     import View
+from users.models     import User
+from users.validators import check_email, check_password
```

Comment:
```suggestion
import json

from django.http      import JsonResponse, request
from django.views     import View

from users.models     import User
from users.validators import check_email, check_password
```
빌트인, 외부 모듈, 사용자 모듈 순으로 모듈을 적고, 다른 성격의 모듈을 적을 때는 공백을 추가해 주세요.

Reply:
빌트인, 외부모듈, 사용자 모듈 순으로 정렬 및 다른 성격의 모듈일때 공백 추가하였습니다.